### PR TITLE
add fetch-all-csv command to collector

### PR DESCRIPTION
**Pull request by cristiandean**
This PR makes it possible to download and extract the CSV files using the following command:

```bash
make build-collector
./collector fetch-all-csv
```

### CODE REVIEW

File: csv-collector/cmd/fetch_csv.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+	"net/http"
+	"os"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+const (
+	firstYear = 2014
+	lastYear  = 2021 // TODO pegar de forma dinâmica o último ano suportado
```
Comment (marcelometal) [inline csv-collector/cmd/fetch_csv.go]: the correct year is `2020`

File: csv-collector/cmd/fetch_csv.go [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+)
+
+var fetchCSVFilesCmd = &cobra.Command{
+	Use:   "fetch-all-csv",
+	Short: "Fetch all CSV files from TSE",
+	Run:   fetchCSVFiles,
+}
+
+func fetchCSVFiles(cmd *cobra.Command, args []string) {
+	for year := firstYear; year < lastYear; year++ {
```
Comment (marcelometal) [inline csv-collector/cmd/fetch_csv.go]: Please, change it to :

```golang
for year := firstYear; year <= lastYear; year++ {

File: csv-collector/cmd/fetch_csv.go (hunk 1 above)
Comment (marcelometal) [reply csv-collector/cmd/fetch_csv.go]: Maybe we can crawling the site bellow to get the correct "last year":
https://www.tse.jus.br/eleicoes/estatisticas/repositorio-de-dados-eleitorais-1/repositorio-de-dados-eleitorais

Iframe=> https://www.tse.jus.br/hotsites/pesquisas-eleitorais/candidatos.html

**Review by marcelometal — CHANGES_REQUESTED** [review]

File: csv-collector/cmd/fetch_csv.go (hunk 2 above)
Comment (cristiandean) [reply csv-collector/cmd/fetch_csv.go]: done!

File: csv-collector/cmd/fetch_csv.go (hunk 1 above)
Comment (cristiandean) [reply csv-collector/cmd/fetch_csv.go]: Crawling can be fragile if the page is changed. Another possibility is the use of `time.Now().Year()` for lastYear and check status code to verify the zip file availability. What do you think?

Comment (marcelometal) [reply csv-collector/cmd/fetch_csv.go]: ok, go ahead! use `time.Now().Year()`

File: csv-collector/cmd/fetch_csv.go [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+		if year%2 != 0 {
+			continue
+		}
+
+		zipFileURL := fmt.Sprintf(tseURL, year)
+		zipFilePath := fmt.Sprintf("./consulta_cand_%d.zip", year)
+		destinationPath := fmt.Sprintf("./consulta_cand_%d", year)
+
+		// Download zip file
+		log.Info(fmt.Sprintf("Downloading %s", zipFileURL))
```
Comment (marcelometal) [inline csv-collector/cmd/fetch_csv.go]: Please, replace all `log.Info` with `log.Infof`

```golang
log.Infof("Downloading %s", zipFileURL)
```

` `

Comment (cristiandean) [reply csv-collector/cmd/fetch_csv.go]: Good catch! Done.

File: csv-collector/cmd/fetch_csv.go (hunk 1 above)
Comment (cristiandean) [reply csv-collector/cmd/fetch_csv.go]: Done!

**Pull request merged**